app/views/layout.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Chitter</title>
+  </head>
+
+  <body>
+    <% if current_user %>
```

Comment:
It feels like there are two responsibilities in this template.  Could you use partials to delegate these responsibilities?